superduperdb/cli/serve.py

```diff
@@ -5,18 +5,32 @@
 
 
 @command(help='Start local cluster: server, ray and change data capture')
-def local_cluster(action: str, notebook_token: t.Optional[str] = None):
+def local_cluster(
+    action: str,
+    notebook_token: t.Optional[str] = None,
+    num_workers: int = 0,
```

Comment:
Should the `num_workers` default to 1?

Reply:
This is something we worked out - in local testing, the head node doubles as a worker node. It can host multiple jobs. So for that `num_workers=0`.